ESP/tools/createzip.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+Import("env")
+
+import sys
+from ntpath import basename
+from zipfile import ZipFile
+import json
+
+
+def createZip(source, target, env):
+    if (
+        sys.platform.startswith("ubuntu")
+        or sys.platform.startswith("Linux")
+        or sys.platform.startswith("linux")
+    ):
```

Comment:
A neat thing about startswith is that it can accept either a string or a tuple of positions to check, so we can simplify this to:

```
if sys.platform.lower().startswith(("ubuntu", "linux")):
    ...
```

Reply:
Love it - definitely a cleaner option.